bloom-housing/bloom
# fix: add geocoding radius functionality

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

This PR includes two recent changes from the old backend:
- Geocoding radius functionality - https://github.com/bloom-housing/bloom/pull/3718
- Email send change - https://github.com/bloom-housing/bloom/pull/3731

## How Can This Be Tested/Reviewed?

The geocoding functionality can be fully tested end to end with the following steps:
1. Create a new preference in the partner site that has geocoding enabled and radius selected and add it to an open listing
2. Go to the public site and apply to the listing - make sure you fill out the address for that preference.
3. Check the DB for that application and the preference extra data should have accurate information (true/false/Unknown)

The email send cannot be tested since the listing changes are still in a different PR.

## Checklist:

- [n/a] I have reviewed the changes in a desktop view
- [n/a] I have reviewed the changes in a mobile view
- [n/a] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
